dnf-behave-tests/features/microdnf/install2.feature

```diff
@@ -0,0 +1,19 @@
+@no_installroot
```

Comment:
I know it's not easy since you're forced to split the feature file, but the names `install[1-8].feature` aren't great. They don't really tell what is inside. e.g. you've got a test for `--setopt=tsflags=nodocs` here in `install2.feaure` and `--nodocs` in `install8.feature`, while they're obviously related.

We should at least try to make it a bit less messy... come up with some better naming scheme?

(unless @m-blaha has a positive update on running the scenarios in separate containers, which would save us from this?)